tiny_dnn/layers/max_unpooling_layer.h

```diff
@@ -151,7 +148,7 @@ class max_unpooling_layer : public feedforward_layer<Activation> {
       in2outmax_;  // mapping max_index(out) => in (1:1)
   };
 
-  std::vector<max_unpooling_layer_worker_specific_storage>
+  max_unpooling_layer_worker_specific_storage
     max_unpooling_layer_worker_storage_;
```

Comment:
@beru By current design, it should be shifted to backend class. But that's a thing for the future, for now I'll make the name shorter.